sunpy/map/sources/soho.py

```diff
@@ -11,7 +11,7 @@
 from matplotlib import colors
 
 from astropy.units import Quantity
-from astropy.visualization import PowerStretch
```

Comment:
Why are you importing LinearStretch if you then don't use it?

Reply:
removed.